src/FlexView.tsx

```diff
@@ -22,7 +22,7 @@ function some(array: any[], predicate: (v: any) => boolean): boolean {
   return array.filter(predicate).length > 0;
 }
 
-type DivProps = Omit<React.HTMLProps<HTMLDivElement>, "ref">;
+type ElementProps = Omit<React.HTMLProps<HTMLElement>, "ref">;
```

Comment:
is there an easy way to make this generic? i.e. have the correct props based on the element passed